Makefile

```diff
@@ -3,59 +3,66 @@ LOCALEPREFIX ?= $(PREFIX)/share/locale
 MANPREFIX    ?= $(PREFIX)/share/man
 PANDOC       ?= $(shell which pandoc)
 XDGPREFIX    ?= /etc/xdg
+SCRIPTS      ?= $(shell find ./bin -type f -executable -printf "%f\n")
+MANPAGES     ?= $(shell find ./doc -type f -name "*.md")
 
 .PHONY: setup
 setup:
 	command -v cram   || aurget cram
 	command -v vbump  || aurget vbump-git
 	command -v pandoc || stack install pandoc
 
-locale/downgrade.pot: bin/downgrade
+locale/%.pot: bin/%
 	xgettext \
 		--from-code=utf-8 -L shell \
-		--package-name=downgrade \
+		--package-name=$* \
 		--copyright-holder=$(AUTHOR) \
 		-o $@ $<
-	find ./locale/*po -exec msgmerge --update {} $@ \;
+	find ./locale/$* -name "*.po" -exec msgmerge --update {} $@ \;
 
 .PHONY: locale
-locale: locale/downgrade.pot
+locale: $(SCRIPTS:%=locale/%.pot)
 
-doc/downgrade.8: doc/downgrade.8.md
+doc/%: doc/%.md
 	$(PANDOC) --standalone --to man $< -o $@
 
 .PHONY: man
-man: doc/downgrade.8
+man: $(MANPAGES:%.md=%)
 
 .PHONY: test
 test:
 	cram test
 
 .PHONY: install
 install:
-	install -Dm755 bin/downgrade $(DESTDIR)$(PREFIX)/bin/downgrade
-	install -Dm755 bin/pacignore $(DESTDIR)$(PREFIX)/bin/pacignore
 	install -Dm644 conf/downgrade.conf $(DESTDIR)$(XDGPREFIX)/downgrade/downgrade.conf
 	install -Dm644 doc/downgrade.8 $(DESTDIR)$(MANPREFIX)/man8/downgrade.8
-	install -Dm644 completion/bash $(DESTDIR)$(PREFIX)/share/bash-completion/completions/downgrade
-	install -Dm644 completion/zsh $(DESTDIR)$(PREFIX)/share/zsh/site-functions/_downgrade
-	install -Dm644 completion/fish $(DESTDIR)$(PREFIX)/share/fish/vendor_completions.d/downgrade.fish
-	for po_file in locale/*.po; do \
+	install -Dm644 doc/pacignore.8 $(DESTDIR)$(MANPREFIX)/man8/pacignore.8
+	for script in $(SCRIPTS); do \
+	  install -Dm755 bin/$$script $(DESTDIR)$(PREFIX)/bin/$$script; \
+	  install -Dm644 completion/$$script/bash $(DESTDIR)$(PREFIX)/share/bash-completion/completions/$$script; \
+	  install -Dm644 completion/$$script/zsh $(DESTDIR)$(PREFIX)/share/zsh/site-functions/_$${script}; \
+	  install -Dm644 completion/$$script/fish $(DESTDIR)$(PREFIX)/share/fish/vendor_completions.d/$${script}.fish; \
+	done
```

Comment:
Two questions on (un)installation targets:

1. Do you think this for-loop over `$(SCRIPTS)` is fine or should we also write this out line-by-line (as we did for the mandocs)?
2. Should we quote all paths in `Makefile` targets?